### PR TITLE
Add org.tabos.saldo

### PR DESCRIPTION
**Pull request by janbrummer**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: CONTRIBUTING.md
Comment (hfiguiere) [inline CONTRIBUTING.md:1]: this shouldn't be here.

File: COPYING
Comment (hfiguiere) [inline COPYING:1]: this shouldn't be here

File: README.md
Comment (hfiguiere) [inline README.md:1]: this shouldn't be here

**Comment by hfiguiere** [comment]
bot, build org.tabos.saldo

File: CONTRIBUTING.md
Comment (janbrummer) [reply CONTRIBUTING.md:1]: Sorry, removed.

File: COPYING
Comment (janbrummer) [reply COPYING:1]: Sorry, removed.

File: README.md
Comment (janbrummer) [reply README.md:1]: Sorry, removed.

**Comment by janbrummer** [comment]
Build fixed.

**Comment by hfiguiere** [comment]
bot, build org.tabos.saldo

**Comment by janbrummer** [comment]
Ok, this build error does not occur locally:   ModuleNotFoundError: No module named 'setuptools_rust'
  error: subprocess-exited-with-error

Anyone here with an idea how to fix it?

File: python3-requirements.json [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/49/86/c0baa3dcac450f784f776e5f245805da14c12f5caa26bfbb03710eb46582/mt_940-4.30.0-py2.py3-none-any.whl",
+                    "sha256": "66fa0c1e942478fd7970b791be017b1ca46556f54d44115008f3982e97ae0a9b"
+                }
+            ]
+        },
+        {
+            "name": "python3-cryptography",
```
Comment (bbhtt) [inline python3-requirements.json:95]: You need to:

1. Add the rust sdk to the manifest
2. Download the cryptography sdist tarball, extract and go to src/rust. Generate the cargo sources file with flatpak-cargo-generator (https://github.com/flatpak/flatpak-builder-tools). Then commit that file and set the necessary rust env vars for the build.
3. Add all build dependencies of cryptography: setuptools_rust (use flatpak pip generator from above to generate it)
4. python-cffi needs to built and installed before the cryptography build starts. Add it seperately before cryptography. Otherwise cryptography's hazmat bindings won't be build (but it's not a build time error, you'll see issues when running)

See https://github.com/flathub/io.github.hydrusnetwork.hydrus/blob/68e4120f51ae9411488fe46f853dc62820c97aed/python3-sources.json#L480-L555 as an example.

File: org.tabos.saldo.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "build-options" : {
+        "env" : {
+            "PIP_PREFIX" : "/app"
```
Comment (bbhtt) [inline org.tabos.saldo.json:28]: Why is this needed? pip genrator adds `prefix={FLATPAK_DEST}` to everything.

File: python3-requirements.json (hunk 1 above)
Comment (janbrummer) [reply python3-requirements.json]: Thanks!! I've added a locally tested version. This was a challenging task :)

**Comment by janbrummer** [comment]
bot, build org.tabos.saldo

**Comment by janbrummer** [comment]
Tested and works great. :-)
